staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -261,8 +261,7 @@ func (wc *watchChan) processEvent(wg *sync.WaitGroup) {
 				continue
 			}
 			if len(wc.resultChan) == outgoingBufSize {
-				klog.V(3).Infof("Fast watcher, slow processing. Number of buffered events: %d."+
-					"Probably caused by slow dispatching events to watchers", outgoingBufSize)
+				klog.V(3).InfoS("Fast watcher, slow processing. Probably caused by slow dispatching events to watchers", "Outgoing bufferedEvents", outgoingBufSize)
```

Comment:
```suggestion
				klog.V(3).InfoS("Fast watcher, slow processing. Probably caused by slow dispatching events to watchers", "outgoingEvents", outgoingBufSize)
```

Reply:
Please use [lowerCamelCase](https://en.wiktionary.org/wiki/lowerCamelCase)  as described in migration instructions.